libbeat/docs/release.asciidoc

```diff
@@ -8,6 +8,15 @@ This section summarizes the changes in each release. Also read
 <<breaking-changes>> for more detail about changes that affect
 upgrade.
 
+* <<release-notes-7.5.1>>
+* <<release-notes-7.5.0>>
+* <<release-notes-7.4.1>>
+* <<release-notes-7.4.0>>
+* <<release-notes-7.3.2>>
+* <<release-notes-7.3.1>>
+* <<release-notes-7.3.0>>
+* <<release-notes-7.2.1>>
+* <<release-notes-7.2.0>>
```

Comment:
Do we have all 6.x patch relases in this list as well?

Reply:
I added the 6.8.x releases.  Good catch.... :)